axelrod/player.py

```diff
@@ -241,7 +241,8 @@ def clone(self):
         return new_player
 
     def reset(self):
-        """Resets history.
+        """
+        Resets history.
         When creating strategies that create new attributes then this method
         should be re-written (in the inherited class) and should not only reset
         history but also rest all other attributes.
```

Comment:
Sorry @MariosZoulias, this has turned up in this PR because I put a line break on line 244, the text wasn't changed by either of us but if you could make the improvements suggested by @marcharper that would be helpful. This file is in `Axelrod/axelrod/player.py`.